runtime/src/main/java/org/corfudb/runtime/clients/BaseHandler.java

```diff
@@ -41,38 +49,18 @@
      * The handler and handlers which implement this client.
      */
     @Getter
+    @Deprecated
     public ClientMsgHandler msgHandler = new ClientMsgHandler(this)
             .generateHandlers(MethodHandles.lookup(), this);
```

Comment:
Can you remove this now?

Reply:
Why do we still need the handleNack, handleWrongEpoch and handleNotReady methods?

Reply:
Other servers not yet implemented with Protobuf messages can still send these, like the ManagementServer, for example. We can only remove them at the very end.